tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
[curious] Should we add any documentation to the `README` regarding TypeScript support and how to use it in MFEs (e.g., does each MFE need to define a `tsconfig.json` file)?

Reply:
Good call, added to readme.rst